doc/source/contributing/how_to_start.md

```diff
@@ -0,0 +1,16 @@
+(how_to_start)=
+# How to start
+
+For adding new docs or guides, we follow the [diataxis](https://diataxis.fr/) framework. Let's suppose you want to write a new guide and not sure where to place it in ArviZ docs, you can check the diataxis' four modes of documentation and place it accordingly.
+
+The audience of ArviZ docs are its users and contributors. Contributing section is for contributors, sand user guide is for users. While adding the new doc files, follow the given points:
+
+* Maintain the hierarchy of headings. There should be only one level one heading in one file, that will be the title of the page.
+* Add proper explanation of the thing you want to explain, it should not be too long ot too short. Try to add the links of the external resources where the basic things are already exlained instead of adding all the explanation yourself.
+* Add table of content tree in every page, see {ref}`syntax_guide`.
+* It is really important to add references. Add cross references to the ArviZ functions and objects, and external libraries.
+* If you are refering to any research paper, or some website, make sure to add its hyperlink.
+* Try to add the examples in the explanation. If it is a code example, add one liner or explain it a little bit before every example. Try to cover all the possible scenarios or if they are a lot, just cover the important ones.
+* Highlight the keywords using backticks.
```

Comment:
These also feel more in the scope of the syntax guide than this page, and are generally covered there already.